src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -5311,6 +5312,9 @@ int RGWRados::restore_obj_from_cloud(RGWLCCloudTierCtx& tier_ctx,
       attrs[RGW_ATTR_CLOUD_TIER_CONFIG] = t_tier;
     }
 
+    // TODO : check whether log_op need to passed to this function
+    log_op = false;
```

Comment:
what needs to be done here?

Reply:
Since it was decided not to sync temporary copies, log_op should be false here.

Reply:
@cbodley please share your thoughts about the same